core/frontend/src/components/autopilot/AutopilotUpdater.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <span />
+</template>
+
+<script lang="ts">
+import axios from 'axios'
+import Vue from 'vue'
+import { getModule } from 'vuex-module-decorators'
+
+import AutopilotStore from '@/store/autopilot'
+import NotificationStore from '@/store/notifications'
+import { autopilot_service } from '@/types/frontend_services'
+import { LiveNotification, NotificationLevel } from '@/types/notifications'
+import { callPeriodically } from '@/utils/helper_functions'
+
+const notification_store: NotificationStore = getModule(NotificationStore)
+const autopilot_store: AutopilotStore = getModule(AutopilotStore)
+
+/**
+ * Responsible for updating autopilot-related data.
```

Comment:
autopilot manager data ?

Reply:
Agree. AutopilotStore sounds like it will have all Autopilot(MAVLink) data